app/controllers/devise/devise_authy_controller.rb

```diff
@@ -112,11 +112,9 @@ def POST_disable_authy
     redirect_to after_authy_disabled_path_for(resource)
   end
 
+  # The verify_authy_installation endpoints are for verification on registration. Verification
+  # on login is handled by the verify_authy methods (the ones without 'installation' in the name).
   def GET_verify_authy_installation
-    if resource_class.authy_enable_qr_code
-      response = Authy::API.request_qr_code(id: resource.authy_id)
-      @authy_qr_code = response.qr_code
-    end
```

Comment:
Do we handle the QR code part of mfa verification elsewhere?

Reply:
@BintLopez Good question: It will be handled in https://github.com/MidwestAccessCoalition/twilio-verify-devise/pull/40. I stopped on that because I wanted to think it over and wanted to get these other changes merged first. But if you'd like to hold off on that one until the QR code one is added, that's cool too!

Reply:
As long as we don't lose the placeholders to that functionality or the blueprints for testing that functionality, I'm fine with addressing that separately/later